slang-antlr/src/main/antlr4/com/sonarsource/slang/parser/SLang.g4

```diff
@@ -215,7 +216,24 @@ nativeBlock
   ;
 
 returnExpression
-  :  RETURN statement
+  :  RETURN statement?
+  ;
+
+jumpStatement
+  :  breakStatement
+  |  continueStatement
+  ;
+
+breakStatement
```

Comment:
should they be `breakExpression` and `continueExpression`? (like return and the rest)